cli/terraform/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "region" {
+  description = "The AWS region to create things in."
+  type        = string
+  default     = "us-east-1"
```

Comment:
Is there a way to make this easily changed without requiring user to edit `docker.py` and `compute.py` etc?

Reply:
They could change this line directly. It would completely re-create their entire infrastructure with a 1-line change.

Reply:
But wont that break `get_latest_task_status_aws` and other boto3 calls using hard-coded regions in `compute.py` and `docker.py`

Reply:
yes, you're right, we could do per-agent region config instead and read it in from the config file